### PR TITLE
docs(changelog): add v13.2.0

### PR DESCRIPTION
**Pull request by adnasa**
adds #531

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## [13.1.0](https://github.com/commercetools/merchant-center-application-kit/compare/v13.0.2...v13.1.1) (2019-04-09)
```
Comment (tdeekens) [inline CHANGELOG.md]: > ## [13.1.1](https://github.com/commercetools/merchant-center-application-kit/compare/v13.1.0...v13.1.1) (2019-04-09)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+#### 💅 Type: Enhancement
```
Comment (tdeekens) [inline CHANGELOG.md]: sorry `Type: Fix` I guess. Depending on what we pick.

File: CHANGELOG.md (hunk 1 above)
Comment (adnasa) [reply CHANGELOG.md]: wops 🙈

File: CHANGELOG.md (hunk 2 above)
Comment (adnasa) [reply CHANGELOG.md]: I personally think it's a feature 😬

Comment (qmateub) [reply CHANGELOG.md]: I see it more as a feature rather than a fix TBH 😅

Comment (tdeekens) [reply CHANGELOG.md]: If feature then it is 13.2. The confusion comes from the PR having false tags then.

Comment (adnasa) [reply CHANGELOG.md]: done ccdfb96

Comment (adnasa) [reply CHANGELOG.md]: oops. did a push without `git add` 😆

**Pull request merged**